src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -726,6 +726,13 @@ public void When_IList_TabView()
 			Assert.AreEqual(2, tabView1.TabItems.Count);
 		}
 
+		[TestMethod]
+		public void When_StateTrigger_PropertyPath()
+		{
+			var s = GetContent(nameof(When_StateTrigger_PropertyPath));
+			var r = Windows.UI.Xaml.Markup.XamlReader.Load(s) as UserControl;
+		}
```

Comment:
There's no assertions in your test!

Reply:
Yes, that's the test :) That there's no exception.